Examples/Physics_applications/pulsar/inputs.corotating.3d.PM

```diff
@@ -0,0 +1,98 @@
+interpolation.nox = 3
+interpolation.noy = 3
+interpolation.noz = 3
+
+
+#################################
+############ PLASMA #############
+#################################
+particles.nspecies = 2
+particles.species_names = plasma_e plasma_p
+
+plasma_e.charge = -q_e
+plasma_e.mass = m_e
+plasma_e.injection_style = "NUniformPerCell"
+plasma_e.profile = parse_density_function
+plasma_e.density_function(x,y,z) = "( ((( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))>=r_star) * ((( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))<(r_star+dR)) )*dens"
+plasma_e.num_particles_per_cell_each_dim = 3 3 3
+plasma_e.momentum_distribution_type = parse_momentum_function
+plasma_e.momentum_function_ux(x,y,z) = "( (( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))<r_star)*(-omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/((1.0-(omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/c)^2)^(0.5)) * (y-yc)/(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5)))"
+plasma_e.momentum_function_uy(x,y,z) = "( (( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))<r_star)*(omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/((1.0-(omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/c)^2)^(0.5)) * (x-xc)/(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5)))"
+plasma_e.momentum_function_uz(x,y,z) = "0.0"
+plasma_e.do_continuous_injection = 0
+plasma_e.density_min = 1E0
+
+plasma_p.charge = q_e
+plasma_p.mass = m_e
+plasma_p.injection_style = "NUniformPerCell"
+plasma_p.profile = parse_density_function
+plasma_p.density_function(x,y,z) = "( ((( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))>=r_star)* ((( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))<(r_star+dR)))*dens"
+plasma_p.num_particles_per_cell_each_dim = 3 3 3
+plasma_p.momentum_distribution_type = parse_momentum_function
+plasma_p.momentum_function_ux(x,y,z) = "( (( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))<r_star)*(-omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/((1.0-(omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/c)^2)^(0.5)) * (y-yc)/(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5)))"
+plasma_p.momentum_function_uy(x,y,z) = "( (( (z-zc)*(z-zc) + (y-yc)*(y-yc) + (x-xc)*(x-xc) )^(0.5))<r_star)*(omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/((1.0-(omega*(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5))/c)^2)^(0.5)) * (x-xc)/(((x-xc)*(x-xc) + (y-yc)*(y-yc))^(0.5)))"
+plasma_p.momentum_function_uz(x,y,z) = "0.0"
+plasma_p.do_continuous_injection = 0
+plasma_p.density_min = 1E0
+
+
+#################################
+######### PULSAR SETUP ##########
+#################################
+pulsar.pulsarType = "dead"             # [dead/active]: sets particle injection type
+pulsar.omega_star = 6245.676               # angular velocity of NS (rad/s)
+pulsar.center_star = 90000 90000 90000
+pulsar.R_star = 12.e3                  # radius of NS (m)
+pulsar.B_star = 8.0323e-6                  # magnetic field of NS (T)
+pulsar.dR = 2e3
+pulsar.verbose = 0                     # [0/1]: turn on verbosity for debugging print statements
```

Comment:
We don't need `pulsar.verbose` anymore since I just added that to let us turn on/off the print statements in the old SpinTheStar branch and I think this branch doesn't have them.

Reply:
I though we might need it while debugging the current implementations.